contrib/lang/haskell/packages.el

```diff
@@ -196,8 +195,7 @@
     :if (configuration-layer/layer-declaredp 'company-mode)
     :init
     (progn
-      (add-to-list 'company-backends 'company-ghc)
-      (add-hook 'haskell-mode-hook 'ghc-comp-init))))
+      (add-to-list 'company-backends 'company-ghc))))
```

Comment:
I think we want `(add-to-list 'company-backends (company-mode/backend-with-yas 'company-ghc))` here (to setup yasnippet to work correctly